src/components/Carrousel/Carrousel.tsx

```diff
@@ -31,44 +59,27 @@ const useStyles = makeStyles(theme => ({
 
 const Carrousel: React.FC<{ newbies: Newbie[] }> = ({ newbies }) => {
   const classes = useStyles();
-  const breakpointSm = useMediaQuery<Theme>(theme => theme.breakpoints.down('xs'));
 
   return (
-    <Grid
-      container
-      spacing={2}
-      wrap={breakpointSm ? 'nowrap' : 'wrap'}
-      className={classes.carrousel}>
-      {newbies.map(newbie => {
-        const startDate = new Date(newbie.startDate);
-        return (
-          <Grid key={newbie.id} item>
-            <Link
-              className={classes.link}
-              to={ROUTES.BUDDY_TASKS_LIST.replace(':newbieId', newbie.id)}>
-              <Paper className={classes.carrouselElement}>
+    <Grid container wrap={'wrap'} justify='center' className={classes.carrousel}>
+      {newbies.map(newbie => (
+        <Grid key={newbie.id} item className={classes.carrouselItem}>
+          <Link
+            className={classes.link}
+            to={ROUTES.BUDDY_TASKS_LIST.replace(':newbieId', newbie.id)}>
+            <Paper className={classes.carrouselElement}>
+              <Box className={classes.carrouselElementInfo}>
                 <Avatar
                   progress={getProgressInPercentages(newbie.tasksInfo.buddyProgress)}
                   name={newbie.name}
                   imgSrc={newbie.photo}
-                  position={newbie.position || undefined}></Avatar>
-                {newbie.startDate && (
```

Comment:
Why starting date field was removed? It was defined since the beginning to be shown since it’s crucial information, maybe we should even consider ordering the newbies by starting date.

Reply:
It was a suggestion from @duiliopastorelli, to separate the things and keep on cards only the main data, all other info - on user datails page. In my opinion now it looks good, with less things on the card, but if you feel we need it I can revert it back

Reply:
I guess start date is main data, it's basically one of the main features of the app to easily see when new person is going to join etc.